src/cluster/cluster.go

```diff
@@ -1,51 +1,62 @@
 package cluster
 
 import (
-	"errors"
-	"firecontroller/component"
 	"firecontroller/microcontroller"
 	mc "firecontroller/microcontroller"
 	"firecontroller/utilities"
 	"log"
-	"strconv"
 
 	"github.com/spf13/viper"
 )
 
 //Cluster - This object defines an array of microcontrollers
 type Cluster struct {
-	Name                  string
-	SlaveMicrocontrollers []mc.Microcontroller
-	Master                *mc.Microcontroller
-	Me                    *mc.Microcontroller
+	Name             string
+	Microcontrollers []mc.Microcontroller
+	Me               *mc.Microcontroller
 }
 
 //Config -
 type Config struct {
-	Name                  string `yaml:"Name"`
-	SlaveMicrocontrollers []mc.Config
-	Master                mc.Config
+	Name             string `yaml:"Name"`
+	Microcontrollers []mc.Config
 }
 
 //GetConfig -
 func (c Cluster) GetConfig() (config Config) {
 	config.Name = c.Name
-	config.Master = c.Master.GetConfig()
-	config.SlaveMicrocontrollers = make([]mc.Config, len(c.SlaveMicrocontrollers))
-	for i, micro := range c.SlaveMicrocontrollers {
-		config.SlaveMicrocontrollers[i] = micro.GetConfig()
+	config.Microcontrollers = make([]mc.Config, len(c.Microcontrollers))
+	for i, micro := range c.Microcontrollers {
+		config.Microcontrollers[i] = micro.GetConfig()
 	}
+
 	return
 }
 
+//Master - returns a pointer to the Master micro
+func (c *Cluster) Master() *mc.Microcontroller {
```

Comment:
This function replaces the pointer to master that was formerly a part of cluster object.